pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,157 @@
+{ config
+, fetchurl
+, stdenv
+, aspellWithDicts
+, at_spi2_core ? null
+, atspiSupport ? true
+, bash
+, glib
+, glibcLocales
+, gnome3
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, gtk3
+, hunspell
+, hunspellDicts
+, hunspellWithDicts
+, intltool
+, isocodes
+, libcanberra_gtk3
+, libudev
+, libxkbcommon
+, pkgconfig
+, procps
+, python3
+, wrapGAppsHook
+, xorg
+, yelp
+}:
+
+let
+  customHunspell = hunspellWithDicts [hunspellDicts.en-us];
+in python3.pkgs.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
```

Comment:
It is preferred to move the helper definitions (like version) to the `let` statement in order not to pollute the build environment.